### ISSUE TITLE
Add Support for Standard Plot Types

### ISSUE THREAD

**Issue by okennedy** [VizierDB/vizier-scala#233]
The current plotting tool has a very rigid input format: The input comes from a single table, with the x-axis in one column, and each y series coming from a different column.  This is great for things like grouped bar plots or some types of line charts, but difficult for other types of data, generally requiring a pre-processing step.  A few examples of challenges:

- Charts measuring aggregate values require preprocessing. 
- Scatter plots can't differentiate points based on a grouping attribute... all y coordinates need to be grouped on the corresponding x coordinate.
- Data comes from a single table.  It's not possible to put multiple datasets on the same chart.
- The UI is very focused on line/grouped-bar plots.

In short, it would be useful to have different UIs (whether via different cell types, or possibly a tabbed UI in the plot cell frontend) for different types of charts.  Types of charts I find myself regularly coming up with preprocessing hacks for:

- [x] Box/Whisker Plots
- [x] CDF Plots
- [x] Aggregate Bar Charts (Sum, Count, Avg, Min, Max, etc...) (possibly a subclass of box/whisker)
- [x] Scatter Plots
